gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HttpHook.java

```diff
@@ -302,6 +304,24 @@ public void setMaxWaitQueueSize(Integer maxWaitQueueSize) {
         this.maxWaitQueueSize = maxWaitQueueSize;
     }
 
+    /**
```

Comment:
Should we further describe the timeout here, or link to a detailed description?

Reply:
Will extend it.